### ISSUE TITLE
Add ...Create Imprints on Modeler bodies, not on DesignBodies. The latter makes the script slow. 

### ISSUE THREAD

**Issue by agvarghe** [ansys/pyansys-geometry#628]
### 📝 Description of the feature

Creating imprints is the slowest operation. We were under the impression that we were using the low-level APIs but that doesn't seem to be the case. We wouldn't create design body untill we are ready to create the Design bodies.

**Comment by jrwest440** [comment]
An operation that absolutely needs to execute on Modeler objects and repeatedly, probably should consider using RunScript and executing internally at the SC API level

**Comment by jmabshof** [comment]
@jrwest440 @RobPasMue can we close this issue?

**Comment by RobPasMue** [comment]
Yes - closing. This is just the way things are implemented... and we can't do much about it I believe, unless the user uses runscript
